lib/releaselib.sh

```diff
@@ -667,25 +667,32 @@ release::gcs::locally_stage_release_artifacts() {
     if [[ -d "$release_stage/server/$platform" ]]; then
       src="$release_stage/server/$platform/$release_kind/server/bin/*"
     fi
-    release::gcs::stage_and_hash $gcs_stage "$src" "$dst" || return 1
+    release::gcs::prepare_tarball $gcs_stage "$src" "$dst" || return 1
 
     # Upload node binaries if they exist and this isn't a 'server' platform.
     if [[ ! -d "$release_stage/server/$platform" ]]; then
       if [[ -d "$release_stage/node/$platform" ]]; then
         src="$release_stage/node/$platform/$release_kind/node/bin/*"
-        release::gcs::stage_and_hash $gcs_stage "$src" "$dst" || return 1
+        release::gcs::prepare_tarball $gcs_stage "$src" "$dst" || return 1
       fi
     fi
   done
 
   logecho "- Hashing files in ${gcs_stage##$build_output/}..."
   find $gcs_stage -type f | while read path; do
-    common::md5 $path > "$path.md5" || return 1
-    common::sha $path 1 > "$path.sha1" || return 1
-    common::sha $path 512 > "$path.sha512" || return 1
+    common::sha $path 256 "full" > "$path.sha256" || return 1
+    common::sha $path 512 "full" > "$path.sha512" || return 1
+  done
+
+  logecho "- Writing artifact hashes to SHA256SUMS/SHA512SUMS files..."
+  for sha_file in "$gcs_stage"/*.sha256; do
+    cat "$sha_file" >> "$gcs_stage/SHA256SUMS"
   done
-}
 
+  for sha_file in "$gcs_stage"/*.sha512; do
+    cat "$sha_file" >> "$gcs_stage/SHA512SUMS"
+  done
+}
```

Comment:
Instead of that we could just do something like:

```sh
  local sum bits
  find "$gcs_stage" -type f | while read -r path; do
    for bits in 256 512; do
      sum="$( common::sha "$path" "$bits" "full" )" || return 1
      echo "$sum" >> "${gcs_storage}/SHA${bits}SUMS"
      echo "$sum" > "${path}.sha${bits}"
    done
  done
```